docker-compose.yml

```diff
@@ -163,6 +163,24 @@ services:
             - ./docker/haproxy/cors.lua:/etc/haproxy/cors.lua
         scale: -1
 
+    loki:
+      image: grafana/loki:2.4.0
+      volumes:
+        - ./docker/loki:/etc/loki
+        - ${CEPH_REPO_DIR}/build.ceph/out:/etc/logs
+      ports:
+        - "3100:3100"
+      command: '--config.file=/etc/loki/loki-config.yaml'
+
+    promtail:
+      image: grafana/promtail:2.4.0
+      volumes:
+        - /var/log:/var/log
+        - ./docker/promtail:/etc/promtail
+        - ${CEPH_REPO_DIR}/build.ceph/out:/etc/logs
+      command: '--config.file=/etc/promtail/promtail-config.yaml'
```

Comment:
Not sure if our version of docker-compose supports it, but what about using [`depends-on:`](https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on) to make promtail run when loki is launched (or the other way around)?

Reply:
Our current version supports it:
https://docs.docker.com/compose/compose-file/compose-file-v2/#depends_on